clients/python/Makefile

```diff
@@ -86,6 +86,10 @@ install-dev: _check_venv_active ## install packages for development
 install-test: _check_venv_active install-dev ## install packages for testing client
 	pip install -r $(PYTHON_DIR)/requirements/test.txt
 
+.PHONY: install-doc
+install-test: _check_venv_active install-dev ## install packages for generating documentation
```

Comment:
`install-test` or `install-doc`? i think the latter, right?

Reply:
Good point. Thanks.
I have corrected this